artman/cli/support.py

```diff
@@ -146,3 +145,27 @@ def select_git_repo(git_repos, target_repo):
     return git_repos['staging']
 
 
+def check_docker_requirements(docker_image):
+    """Checks whether all docker-related components have been installed."""
+
+    try:
+        subprocess.check_output(['docker', '--version'])
+    except OSError:
+        logger.error(
+            'Docker not found on path or is not installed. Refert to '
+            'https://docs.docker.com/engine/installation about how to install '
+            'Docker on your local machine.')
+        sys.exit(1)
```

Comment:
Nitpick: Use something other than `1` as the exit code here, since you will likely want to bubble the exit code from gradlew when you run it, and that is likely to often be `1`.